akka-cluster/src/main/resources/reference.conf

```diff
@@ -196,6 +196,52 @@ akka {
       expected-response-after = 1 s
 
     }
+    
+    # TODO name to be determined
+    multi-datacenter {
```

Comment:
The property that Johan is adding is named `cross-data-center-connections` so perhaps this should be `cross-data-center-failure-detector`.

Or does it deserver a new section?
```
multi-data-center {
  belongs-to-data-center = "default"
  cross-failure-detector {}
  cross-connections = 5
}
```

?